src/components/MessageInput/MessageInput.tsx

```diff
@@ -182,19 +182,17 @@ export const MessageInputWithContext = <
 ) => {
   const {
     additionalTextInputProps,
-    appendText,
```

Comment:
with props like this that were removed as they're not used in this component directly anymore, also remove them from the types that are on these functions. For instance, I see `pickFile` below was removed on line 208 but still exists on the type on line 148. Don't want to allow that to be passed in as a type if this component isn't going to be using it

Reply:
Done!! Have cleaned up few more types, missed from previous PRs :)